Source/WebCore/Modules/mediastream/RTCPeerConnection.cpp

```diff
@@ -577,6 +577,9 @@ bool RTCPeerConnection::doClose()
     }
     m_operations.clear();
 
+    for (auto& transport : m_dtlsTransports)
+        transport->close();
+
```

Comment:
Looks like you have two changes in this PR

Reply:
Gasp, I pushed too eagerly!, will fix this shortly